src/Microsoft.ServiceFabric.Actors/Remoting/V2/FabricTransport/Runtime/FabricTransportActorServiceRemotingListener.cs

```diff
@@ -37,13 +37,15 @@ public class FabricTransportActorServiceRemotingListener : FabricTransportServic
         /// <param name="listenerSettings">
         ///     The settings to use for the listener.
         /// </param>
-        /// <param name="exceptionConvertors">Convertors to convert user exception to service exception.</param>
-        /// <param name="requestForwarderFactory">Request forwarder incase migration is ongoing and current service cannot service the request.</param>
-        public FabricTransportActorServiceRemotingListener(
-            ActorService actorService,
-            FabricTransportRemotingListenerSettings listenerSettings = null,
-            IEnumerable<IExceptionConvertor> exceptionConvertors = null,
-            Func<RequestForwarderContext, IRequestForwarder> requestForwarderFactory = null)
+        /// <param name="exceptionConvertors">Convertors to convert user exception to service exception.</param>
+        /// <param name="requestForwarderFactory">Request forwarder incase migration is ongoing and current service cannot service the request.</param>
+#pragma warning disable CS0618 // Type or member is obsolete
+        public FabricTransportActorServiceRemotingListener(
```

Comment:
For public APIs that use types, like IRequestForwarder, that were just made obsolete, make them obsolete instead of suppressing CS0618 like this. For each method like this, add a new overload without obsolete parameters.